'R: Push Release Notes' failing

https://github.com/checkstyle/checkstyle/actions/runs/3559572917/jobs/5979018889

```
Run ./.ci/update-releasenotesxml.sh 10.5.0
TARGET_VERSION=10.5.0
CURRENT_VERSION=10.5.0
Error: Define READ_ONLY_TOKEN environment variable
Error: Process completed with exit code 1.
```

--- a/.github/workflows/update_sources.yml
+++ b/.github/workflows/update_sources.yml
@@ -20,6 +20,8 @@ jobs:
         run: |
           sudo apt-get install xmlstarlet
       - name: Run Shell Script
+        env:
+          READ_ONLY_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           ./.ci/update-releasenotesxml.sh ${{ github.event.inputs.version }}
       - name: Commit and Push

